src/main/main.scss

```diff
@@ -0,0 +1,80 @@
+@import './variables';
+
+body,
+html {
+  height: 100%;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  width: $stopwatchSize;
+  height: $stopwatchSize;
+  border: 1vmin dotted $borderColor;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__center {
+    position: absolute;
+    width: $stopwatchCenterSize;
+    height: $stopwatchCenterSize;
+    background-color: $backgroundColorCenter;
+    border-radius: 50%;
+    z-index: 2;
+  }
+
+  &__hand {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform-origin: bottom;
+    transform: translateX(-50%) translateY(-100%);
+
+    &--seconds {
+      width: 1.5vmin;
+      height: 38vmin;
+      background-color: $backgroundColorSeconds;
+      animation: rotate-seconds 60s linear infinite;
+    }
+
+    &--minutes {
+      width: 3vmin;
+      height: 20vmin;
+      background-color: $backgroundColorMinutes;
+      animation: rotate-minutes 3600s steps(60) infinite;
+    }
+  }
+
+  &--speed-up {
+    & .stopwatch__hand--seconds {
+      animation-duration: 10s;
+    }
+
+    & .stopwatch__hand--minutes {
+      animation-duration: 600s;
+    }
+  }
+}
+
+@keyframes rotate-seconds {
```

Comment:
```suggestion
@keyframes rotate{
```

Reply:
Sure! My mistake. They are the same.🤦🏻‍♀️